### PR TITLE
feat(FOROME-1425): Reworking wizard scenarios

### PR DESCRIPTION
**Pull request by AndreyUstyumenko**
https://quantori.atlassian.net/browse/FOROME-1425

### CODE REVIEW

**Review by Lapk1n — COMMENTED** [review]
1. Open whole genome for xl_PGP3140_wgs_NIST-4_2
2. Edit choice for "Start with card"
3. Click continue with the whole genome option
4. Go back by the wizard header icon (edit choice is visible)

File: src/pages/main/components/selected-dataset/build-flow/components/cards/description-card/description-card.tsx [hunk 1]
```diff
@@ -73,16 +73,17 @@ export const DescriptionCard = observer(
...
         <CardTitleWithEdit
           title={ds}
-          isEditShown={editDisabled}
+          isEditShown={!editDisabled}
           onEdit={() => wizardStore.editCard(id)}
         />
 
         <div className="mb-4">
-          <Card className="w-full mt-4 bg-grey-tertiary">
+          <Card className="w-full mt-4 bg-grey-tertiary" position={'stretch'}>
```
Comment (QSdmitrioul) [inline src/pages/main/components/selected-dataset/build-flow/components/cards/description-card/description-card.tsx]: position="stretch"

File: src/pages/main/components/selected-dataset/build-flow/components/cards/presets-card/presets-card.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import presetsCardStore from './presets-card.store'
 
 export const PresetsCard = observer((props: ICardProps) => {
```
Comment (QSdmitrioul) [inline src/pages/main/components/selected-dataset/build-flow/components/cards/presets-card/presets-card.tsx]: Seems that its easier do deconstruct props here

File: src/pages/main/components/selected-dataset/build-flow/components/cards/start-card.tsx [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
 import { Card } from '@ui/card'
-import { Radio } from '@ui/radio'
-import { startFlowOptionsList } from '../wizard/wizard.data'
 import { ICardProps } from '../wizard/wizard.interface'
 import wizardStore from '../wizard/wizard.store'
 import { CardTitleWithEdit } from './components/card-edit-title'
+import { useRadioListData } from './components/card-radio.hooks'
+import { CardRadioList } from './components/card-radio-list'
 
 export const StartCard = (props: ICardProps) => {
```
Comment (QSdmitrioul) [inline src/pages/main/components/selected-dataset/build-flow/components/cards/start-card.tsx]: Same here

File: src/pages/main/components/selected-dataset/components/card-explore/card-sections.utils.ts [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
+    location: { protocol, host },
+  } = window
 
-  const url = new URL(
-    `${window.location.protocol}${window.location.host}${location}`,
-  )
+  const url = new URL(`${protocol}${host}${location}`)
   let result = ''
 
   const isFilter = url.pathname === Routes.Refiner
```
Comment (QSdmitrioul) [inline src/pages/main/components/selected-dataset/components/card-explore/card-sections.utils.ts]: switch seems here more suitable(my opinion)

File: src/pages/main/components/selected-dataset/start-flow/selected-dataset-start-flow.tsx [hunk 5]
```diff
@@ -26,13 +28,18 @@ export const SelectedDatasetStartFlow = observer(
...
+          editDisabled={true}
+          id={WizardCardIds.StartFull}
+        />
       </div>
 
       <div className="flex-grow grid gap-4 grid-cols-4 p-4 overflow-y-auto">
         <Card
           className="col-span-2"
           style={{ paddingLeft: 0, paddingRight: 0 }}
+          position={'stretch'}
```
Comment (QSdmitrioul) [inline src/pages/main/components/selected-dataset/start-flow/selected-dataset-start-flow.tsx]: =""

**Comment by AndreyUstyumenko** [comment]
@Lapk1n Wasn't able to reproduce, in my case it rolled back exactly to the correct step.

**Review by Lapk1n — APPROVED** [review]

**Review by QSdmitrioul — APPROVED** [review]

**Pull request merged**